sections/semantics-document-metadata.include

```diff
@@ -410,6 +409,30 @@
     information regarding the license under which the current page is provided.
   </div>
 
+  <{link}> and <{a}> elements may also have a
+  <dfn element-attr for="links,link,a"><code>rev</code></dfn> attribute, which is used to describe
+  a <a>reverse link</a> relationship from the resource specified by the <{links/href}> to the
+  current document. If present, the value of this attribute must be a <a>set of space-separated
+  tokens</a>. Like the <{links/rel}> attribute, [[#sec-link-types]] describes the <a>allowed
+  keywords and their meanings</a> for the <{links/rev}> attribute. Both the <{links/rel}> and
+  <{links/rev}> attributes may be present on the same element.
+
+  <dfn lt="reverse link|Reverse links">Reverse links</dfn> compliment the typical "forward link" by
+  describing the relationship that the linked-to document has with the current document. This can
+  enable user agents to build a more comprehensive map of linked documents.
+
+  <div class="example">Given two documents, each containing a chapter of a book, the links between
+  them could be described with the <{link/rel}> and <{links/rev}> attributes as follows (where both
+  links are semantically equivalent:
+
+  Document with URL "chapter1.html" 
+  <pre highlight="html">&lt;link href="chapter2.html" rel="Next"&gt;</pre>
+  
+  Document with URL "chapter2.html" 
+  <pre highlight="html">&lt;link href="chapter1.html" rev="Prev"&gt;</pre>
+
```

Comment:
Actually, I think the example is wrong.

The relationships defined are **directional**. So if the "Next" document, for doc1, is doc2, then doc2 has a "previous" document which is doc1.

In the example here Chapter1.html would have
```<link rel="Next" rev="Prev" href="Chapter2.html">```
(and Chapter2.html would have
```<link rel="Prev" rev="Next" href="Chapter1.html">```
```<link rel="Next" rev="Prev" href="Chapter3.html">```
by way of symmetry)


A table of contents might have
  <li><a href="ch1" rel="Next" rev="toc">chapter 1</a></li>
  <li><a href="ch2" rev="toc">chapter 1</a></li>
  <li><a href="ch3" rev="toc">chapter 1</a></li>